### PR TITLE
Email queue tree and form view #6415

### PR DESCRIPTION
**Pull request by prakashpp**

### CODE REVIEW

File: view/email_queue_form.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<form string="Email Queued" col="4">
+    <label name="from_addr"/>
+    <field name="from_addr"/>
+    <label name="to_addrs"/>
+    <field name="to_addrs"/>
+    <separator id="msg" string="Message" colspan="4"/>
+    <field name="msg" colspan="4"/>
```
Comment (sharoonthomas) [inline view/email_queue_form.xml]: Whats the point of displaying message ? It is going to be base64 encoded and cloud be really large if there are attachments.

File: view/email_queue_tree.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<tree string="mails queued">
```
Comment (sharoonthomas) [inline view/email_queue_tree.xml]: Email Queue

File: view/email_queue_form.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<form string="Email Queued" col="4">
```
Comment (sharoonthomas) [inline view/email_queue_form.xml]: Email Queue

File: email_queue.xml [hunk 4]
```diff
@@ -1,6 +1,67 @@
...
+            <field name="view" ref="email_queue_view_tree"/>
+            <field name="act_window" ref="act_email_queue_form"/>
+        </record>
+        <record model="ir.action.act_window.domain" id="act_email_queue_form_domain_outbox">
+            <field name="name">Outbox</field>
+            <field name="sequence" eval="10"/>
+            <field name="domain">[('state', '=', 'outbox')]</field>
+            <field name="act_window" ref="act_email_queue_form"/>
+        </record>
+        <record model="ir.action.act_window.domain" id="act_email_queue_form_domain_sending">
```
Comment (sharoonthomas) [inline email_queue.xml]: unlikely to have any records

Comment (prakashpp) [inline email_queue.xml]: agree

File: view/email_queue_form.xml (hunk 3 above)
Comment (prakashpp) [inline view/email_queue_form.xml]: done

File: view/email_queue_form.xml (hunk 1 above)
Comment (prakashpp) [inline view/email_queue_form.xml]: didnt think about base64

File: view/email_queue_tree.xml (hunk 2 above)
Comment (prakashpp) [inline view/email_queue_tree.xml]: done

**Pull request merged**